src/tests/class_tests/openms/source/IdentificationDataConverter_test.cpp

```diff
@@ -86,16 +94,23 @@ START_SECTION((void importIDs(IdentificationData&, const vector<ProteinIdentific
                true);
   }
 
+  std::sort(proteins_in.begin(), proteins_in.end(), ComparePIdSize());
+  std::sort(proteins_out.begin(), proteins_out.end(), ComparePIdSize());
```

Comment:
sort the results before comparing -> now they should be in the same order and should be equal